sqlmesh/utils/migration.py

```diff
@@ -0,0 +1,14 @@
+from sqlglot.dialects.dialect import DialectType
+
+# This is an ad-hoc upper bound which doesn't necessarily match MySQL's
+# TEXT storage specification, but it should be good for most use cases.
+MAX_TEXT_LENGTH = 1024
+
+
+def primary_key_text_type(dialect: DialectType) -> str:
```

Comment:
change the name of this, it's not primary key, it's index

Reply:
Done